compiler/x/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "codegen/AheadOfTimeCompile.hpp"
+#include "env/FrontEnd.hpp"
+#include "codegen/Instruction.hpp"
+#include "compile/Compilation.hpp"
+#include "compile/ResolvedMethod.hpp"
+#include "runtime/RelocationRecord.hpp"
+#include "compile/VirtualGuard.hpp"
+#include "env/CompilerEnv.hpp"
+#include "env/jittypes.h"
+#include "il/Node.hpp"
+#include "il/Node_inlines.hpp"
+#include "il/SymbolReference.hpp"
+#include "ras/DebugCounter.hpp"
+#include "runtime/CodeCacheConfig.hpp"
+#include "runtime/CodeCacheManager.hpp"
+#define NON_HELPER   0x00
```

Comment:
Why is this defined?  Who uses it?

Reply:
Thank you, I believe this was a copying artifact. Shipping the corrected changes.